springwolf-core/src/test/java/io/github/stavshamir/springwolf/asyncapi/DefaultAsyncApiSerializerServiceTest.java

```diff
@@ -18,20 +18,20 @@
 import lombok.Data;
 import org.apache.commons.io.IOUtils;
 import org.json.JSONException;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+
+import org.junit.jupiter.api.Test;
 import org.skyscreamer.jsonassert.JSONAssert;
 import org.skyscreamer.jsonassert.JSONCompareMode;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.SpringRunner;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
 import java.util.Map;
 
-@RunWith(SpringRunner.class)
```

Comment:
The JUnit5 equivalent would be `@ExtendWith(SpringExtension.class)`. `@SpringBootTest` would work as well, but will make the test much slower